themes/bootstrap5/templates/devtools/icon.phtml

```diff
@@ -2,8 +2,8 @@
   $title = $this->translate('Icons');
   $titleEsc = $this->escapeHtml($title);
   $this->headTitle($title);
-  $this->layout()->breadcrumbs = '<li class="breadcrumb-item"><a href="' . $this->url('devtools-home') . '">Development Tools</a>'
-  . '<li class="breadcrumb-item active" aria-current="page">' . $titleEsc . '</li>';
+  $this->breadcrumbs()->set('Development Tools', $this->url('devtools-home'))
+    ->add($titleEsc, active: true);
```

Comment:
This is a tricky one! $titleEsc is the HTML-escaped version of $title... but since the breadcrumb helper now takes care of escaping for us, we need to change to use the unescaped variable.

```suggestion
    ->add($title, active: true);
```